tomoscan/tomoscan.py

```diff
@@ -534,7 +534,7 @@ def set_exposure_time(self, exposure_time=None):
                 exposure_time = self.epics_pvs['ExposureTime'].value
             self.epics_pvs['CamAcquireTime'].put(exposure_time, wait=True, timeout = 10.0)
 
-    def set_flat_exposure_time(self, exposure_time=None):
+    def set_scan_exposure_time(self, exposure_time=None):
         """Sets the camera exposure time for flat fields.
```

Comment:
@decarlof please update the comment

Reply:
addressed [here](https://github.com/tomography/tomoscan/pull/144/commits/76af9722ed7c6a9be60904c7fc3d392720b38e2f)